### PR TITLE
`2517` copy updates

### LINKED ISSUE #2517: "My Safes" Copy Updates
https://www.notion.so/hulking-cosmos-3e8/Q4-2024-QA-Requirements-13e2ee9c5bc280a88458e7a935d17488?pvs=4#1422ee9c5bc2800886cecc075d2a04ee

“My Safes” => “My DAOs”

### PR DESCRIPTION
**Pull request by Da-Colon**
A Copy Small copy updates

Closes #2517

### CODE REVIEW

**Review by DarksightKellar — CHANGES_REQUESTED** [review]
There're a couple more places the issue points out in the notion desc that haven't been updated in this PR

File: src/i18n/locales/en/navigation.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "dashboard": "Dashboard",
+  "home": "Home",
```
Comment (adamgall) [inline src/i18n/locales/en/navigation.json:2]: This change isn't a part of the linked issue.

File: src/i18n/locales/en/settings.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
   "permissionsTitle": "Permissions",
   "daoSettingsGeneral": "General",
   "daoSettingsGovernance": "Governance",
-  "daoMetadataName": "Safe Name",
+  "daoMetadataName": "DAO Name",
```
Comment (adamgall) [inline src/i18n/locales/en/settings.json:19]: This change isn't a part of the linked issue.

File: src/i18n/locales/en/transaction.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
   "pendingDeploySafe": "Deploying your Safe",
   "failedDeploySafe": "Deployment failed",
   "successDeploySafe": "Deployment successful",
-  "failedIndexSafe": "Your Safe was deployed but is not yet available to view. It has been added to your favorited list of 'My Safes'.",
+  "failedIndexSafe": "Your Safe was deployed but is not yet available to view. It has been added to your favorited list of 'My DAOs'.",
```
[the remark this answers is not in the document]
Comment (adamgall) [reply src/i18n/locales/en/transaction.json:15]: This entire change isn't part of the linked issue.

**Review by adamgall — CHANGES_REQUESTED** [review]
This PR:
- does not cover all of the items in #2517 
- covers extra stuff which is not part of the items in #2517 

For the sake of accurately tracking what is being done, given our source of truth (the Notion doc), please either/or:
- modify the github issues
- modify the links to github issues in the Notion Doc
- add/remove changes from this PR

so that the linked Issue for this PR covers exactly the changes described in the Notion doc which has the github issue link.

**Comment by Da-Colon** [comment]
Like 1 per pr?

**Comment by adamgall** [comment]
@Da-Colon im just saying the issue linked in this PR does not line up with the work that happened in this PR.

**Comment by Da-Colon** [comment]
Cool beans. I'll adjust stuff

**Pull request closed without merging**